### ISSUE TITLE
Act Console should support SSH in prod mode by default instead of TELNET

### ISSUE THREAD

**Issue by CINJ** [actframework/actframework#458]
ACT.Framework should support SSH as well as TELNET, as TELNET is inherently insecure, it poses a potential vulnerability in a production environment which is misconfigured (both within ACT or upstream).

My request is to allow deployment configuration of ACT to use SSH or TELNET, however I would also request that SSH is the default setting for PROD mode to ensure developers and devops are 'secure by default'

**Comment by CINJ** [comment]
@greenlaw110 My concern is that we are using an outdated and insecure technology to run the CLI. Even though the concept is to never expose to external network a small configuration mistake upstream (out of ACT control) can have a huge impact. By implementing SSH (especially by default) this could be easily mitigated. 

There is a privledge escalation issue here as well. If I have an unprivleged account on a server running ACT I cannot access its files, or change the service, however all I need to do is TELNET into it's service and I can 'escalate my privledge' to be able to start stop or admin the service!

A short term solution would be to configure the TELNET service not to respond to anything other than 127.0.0.1 by default when it is installed (in fact this should be the case no matter what - even if using DEV configuration!) - even then, this does not stop the privledge escalation.

**Comment by greenlaw110** [comment]
@CINJ agreed that ip address shall be limited to 127.0.0.1. However expose any additonal ports to external network is not recommended.

With regard to unprileged account, my thought is that should be a premium service running multi-tenant act deployment, which should be addressed in act-commercial project. The idea is to provide a HTML based console for accounts to manage their act service, exactly what Zenexity did for playframework host service.

**Comment by CINJ** [comment]
@greenlaw110, after discussion I think we agree 127.0.0.1 by default and no telnet enabled by default (to require explicit configuration setting)

**Comment by CINJ** [comment]
To consider... 
https://mina.apache.org/sshd-project/
